js_modules/dagit/packages/core/src/asset-graph/AssetRunLogObserver.tsx

```diff
@@ -0,0 +1,157 @@
+import {gql, useSubscription} from '@apollo/client';
```

Comment:
Can we add a test for this file? (ChatGPT can help here :) )

Reply:
Will do! I think having tests for this would also make it easier to apply some of the other refactors suggested here, will set it up 👍